solidity/contracts/Gravity.sol

```diff
@@ -258,14 +251,15 @@ contract Gravity is ReentrancyGuard {
 		// Success
 	}
 
-	function isOrchestrator(ValsetArgs memory _newValset, address _sender) private pure returns(bool) {
+	function isOrchestrator(ValsetArgs memory _valset, address _sender) private pure returns(bool) {
 
-		for (uint256 i = 0; i < _newValset.validators.length; i++) {
-			if(_newValset.validators[i] == _sender) {
+		for (uint256 i = 0; i < _valset.validators.length; i++) {
+			if(_valset.validators[i] == _sender) {
```

Comment:
with the new changes, you are not using the return type, please consider changing the whole function